guides/common/modules/proc_upgrading-external-database.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Upgrading_the_External_Database_{context}"]
+= Upgrading the External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+. On a virtual machine, take a snapshot.
+. On a physical machine, create a backup.
+. Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/using-databases#migrating-to-a-rhel-8-version-of-postgresql_using-postgresql[Migrating to a RHEL 8 version of PostgreSQL].
+
+.Procedure
+. Create a backup.
+. If you keep the same hostname on the new PostgreSQL server, you need to update the DNS preventing the need to do updates on {Project} itself.
```

Comment:
```suggestion
. If you keep the same hostname on the new PostgreSQL server, update the DNS record for the host so that ...
```
The meaning of the second sentence is unclear to me. Can you be more specific?
Is this about updating DNS so that {Project} can still see the host as registered? Or what would happen if I didn't update the DNS?